### PR TITLE
Registration Tables: Tabbed Dashboard

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#13100

*Description of changes:*
* Implements Tab component to house Registration Tables
* Refactors Dashboard to extract PPR Registration Table to it's own Subcomponent for easier Re-use
* Updated unit testing
* Various style updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/dashboard/DashboardTabs.vue [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      <b>Personal Property Registrations</b><span class="pl-1">({{ registrationsCount }})</span>
+    </v-tab>
+    <v-tab
+      tabindex="1"
+      class="tab upper-border"
+      :ripple="false"
+      :class="{ 'mt-1': tabNumber === 0 }"
+    >
+      <v-icon class="mr-2" :class="{'whiteIcon': tabNumber === 0}">mdi-home</v-icon>
+      <b>Manufactured Home Registrations</b><span class="pl-1">(10)</span>
```
Comment (cameron-eyds) [inline ppr-ui/src/components/dashboard/DashboardTabs.vue:26]: Hardcoded mocked count

File: ppr-ui/src/components/dashboard/DashboardTabs.vue [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          @snackBarMsg="snackBarEvent($event)"
+        />
+      </v-tab-item>
+      <v-tab-item class="px-7">
+        <PprRegistrations
+          isTabView
+          :appReady="appReady"
+          :appLoadingData="appLoadingData"
+          @snackBarMsg="snackBarEvent($event)"
+        />
```
Comment (cameron-eyds) [inline ppr-ui/src/components/dashboard/DashboardTabs.vue:43]: Just as a placeholder until MhrRegistrations component is built

**Review by ChasBoucher — APPROVED** [review]
Looks Good.

File: ppr-ui/src/utils/feature-flags.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export const defaultFlagSet: LDFlagSet = {
   'search-registration-number': false,
   'search-serial-number': false,
   'mhr-ui-enabled': false,
-  'mhr-registration-enabled': false
+  'mhr-registration-enabled': true
```
Comment (dimak1) [inline ppr-ui/src/utils/feature-flags.ts]: Are we enabling the MHR reg or it got committed by mistake?

**Review by dimak1 — APPROVED** [review]
Good stuff, just one comment about feature-flag.

File: ppr-ui/src/utils/feature-flags.ts (hunk 3 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/feature-flags.ts]: Definitely a mistake, good catch, thank you! 
Will revert

Comment (dimak1) [reply ppr-ui/src/utils/feature-flags.ts]: 👍🏻

**Pull request merged**